docs/docs/components/service-mesh.md

```diff
@@ -14,6 +14,20 @@ all workloads which have a specified configuration. The service mesh container f
 sets up `iptables` rules based on its configuration and then starts
 [Envoy](https://www.envoyproxy.io/) for TLS origination and termination.
 
+## Service mesh startup enforcement
+Since Contrast doesn't yet enforce the order in which the containers are started
+(see [Limitations](../features-limitations.md)), we deny all incoming connections
+until the service mesh resolves this deny rule.
+A systemd unit inside the podVM creates this deny rule.
+The kata-agent systemd unit requires that this unit successfully runs and exists.
+Therefore, the deny rule is in place, before any containers can be started.
```

Comment:
Maybe this is to technical for the docs? We should rather describe the guarantees and consequences as how this is implemented.

Reply:
Should I just delete the marked sentences? Do you think the docs are sufficient without it?